python/nav/web/crispyforms.py

```diff
@@ -195,3 +195,14 @@ class and template definitions.
         form_id=form_id,
         form_class=form_class,
     )
+
+
+class FormCheckBox:
+    """Checkbox suited for the NAV layout
+
+    :param field: A field to render as a checkbox field.
+    """
+
+    def __init__(self, field):
+        self.field = field
+        self.input_type = 'checkbox'
```

Comment:
I think it is better to place this class with the `CheckBox` class, not at the bottom of the file

Reply:
#3191 #3192